### ISSUE TITLE
AG WG Review: WCAG2ICT adding first set of level AAA success criteria

### ISSUE THREAD

**Issue by pday1** [w3c/wcag2ict#912]
## Summary
The WCAG2ICT Task Force has extended guidance to begin incorporating level AAA success criteria. We have not yet finished, but wanted to get your input at this early stage. 
The work is split into 3 main areas:

1. Comments added for the first group of level AAA success criteria (1.2.6 2.3.3). 
2. Structural change: a separate section has been added for comments on level AAA.  However, we have also included a placeholder link to each level AAA comment within the section containing Comments on Level A and AA success criteria, so the relationship between all the SCs is still apparent (and so level AAA are not overlooked).  
3. Introductory statement has been added to the start of this new comments on level AAA section to explain the importance of level AAA criteria, while also acknowledging the difficulty of satisfying all level AAA criteria.

See Content for Review section for more details and for links to the individual areas for review. 
	
## Maturity Level
Refining

## Action Needed
* Review content (See Content for Review section below for details.)
* Use the thumbs up and thumbs down icons to indicate agreement or disagreement with various comments. Do not add comments to simply support other comments. 
* Note any suggested edits for improvement and/or reasoning either in a comment on the specific issue for that topic, or by creating a PR on the WCAG2ICT document.  

----

## Background information 

Once finalized, this new WCAG2ICT Note will replace the 2013 Note, and it covers all three WCAG versions (2.0, 2.1, or 2.2).  This is important as different jurisdictions are using different versions of WCAG for the accessibility of information and communication technology. 

## Content for Review

Please review the following major changes to the WCAG2ICT editor's draft. If you have comments or concerns with those sections, please add them in the specific issue for that topic. 

1. Comments on level AAA SCs that are currently in the editor's draft are as follows:

    | Content for review | Issue for giving feedback |
    | ---|--- |
    | **Comments on level AAA SCs** |  |
    | 1.2.6 Sign Language (Prerecorded) | [888](https://github.com/w3c/wcag2ict/issues/888) |
    |  1.2.7 Extended Audio Description (Prerecorded) |  [889](https://github.com/w3c/wcag2ict/issues/889) |
    | 1.2.8 Media Alternative (Prerecorded) |  [890](https://github.com/w3c/wcag2ict/issues/890) |
    | 1.2.9 Audio-only (Live) |  [891](https://github.com/w3c/wcag2ict/issues/891)|
    | 1.3.6 Identify Purpose |  [903](https://github.com/w3c/wcag2ict/issues/903) |
    | 1.4.6 Contrast (Enhanced) |  [892](https://github.com/w3c/wcag2ict/issues/892) |
    | 1.4.7 Low or No Background Audio | [893](https://github.com/w3c/wcag2ict/issues/893) |
    | 1.4.8 Visual Presentation |  [894](https://github.com/w3c/wcag2ict/issues/894) |
    | 1.4.9 Images of Text (No Exception) |  [895](https://github.com/w3c/wcag2ict/issues/895) |
    | 2.1.3 Keyboard (No Exception) |  [896](https://github.com/w3c/wcag2ict/issues/896) |
    | 2.2.3 No Timing |  [897](https://github.com/w3c/wcag2ict/issues/897) |
    | 2.2.4 Interruptions |  [898](https://github.com/w3c/wcag2ict/issues/898) |
    | 2.2.5 Re-authenticating |  [899](https://github.com/w3c/wcag2ict/issues/899) |
    | 2.2.6 Timeouts |  [909](https://github.com/w3c/wcag2ict/issues/909) |
    | 2.3.2 Three Flashes |  [910](https://github.com/w3c/wcag2ict/issues/910) |
    | 2.3.3 Animation from Interactions |  [911](https://github.com/w3c/wcag2ict/issues/911) |
    | **Other changes** | |
    |  Structural change: comments on level AAA now in separate section, but linked from main section. |     https://github.com/w3c/wcag2ict/issues/900 |
    |  Introductory statement at start of level AAA section | https://github.com/w3c/wcag2ict/issues/901 |

## Reference Material
None

**Comment by maryjom** [comment]
Since there wasn't an issue on updates to the very beginning sections, I opened a specific issue regarding the "Comparison to..." the previous version of the Note. See #988 for details.

**Comment by maryjom** [comment]
@pday1 I don't see a review issue for the SC Problematic for Closed Functionality section, or is that section still under development and not being included in this AG WG review?

For now, I'll start to gather comments on that section and add them here.
- [1.2.6 Sign Language (Prerecorded)](https://w3c.github.io/wcag2ict/#sign-language-prerecorded) - In the description, "translation" should be "interpretation".

**Comment by pday1** [comment]
> [@pday1](https://github.com/pday1) I don't see a review issue for the SC Problematic for Closed Functionality section, or is that section still under development and not being included in this AG WG review?

Apologies. 
Issue https://github.com/w3c/wcag2ict/issues/995 has been opened for any feedback on Level AAA SCs under the SC problematic for closed functionality section.

**Comment by maryjom** [comment]
@pday1 There was also a missing review issue for 3.1.6 Pronunciation and the line was missing in the content for review table at the top of this issue. I opened issue #1000 and structured it like the rest of the review issues and added it to the table. I had a comment, so will log in that new issue.

**Comment by pday1** [comment]
> [@pday1](https://github.com/pday1) I don't see a review issue for the SC Problematic for Closed Functionality section, or is that section still under development and not being included in this AG WG review?
> 
> For now, I'll start to gather comments on that section and add them here.
> 
> * [1.2.6 Sign Language (Prerecorded)](https://w3c.github.io/wcag2ict/#sign-language-prerecorded) - In the description, "translation" should be "interpretation".

Now moved to issue to track all SC problematic for closed:
https://github.com/w3c/wcag2ict/issues/995
